src/commands/env/env-get.js

```diff
@@ -16,9 +19,21 @@ const envGet = async (name, options, command) => {
     return false
   }
 
-  const siteData = await api.getSite({ siteId })
+  const { siteInfo } = cachedConfig
+  let { env } = cachedConfig
+
+  if (siteInfo.use_envelope) {
+    env = await getEnvelopeEnv({ api, context, env, key: name, scope, siteInfo })
+  } else if (context !== 'dev' || scope !== 'any') {
+    error(
```

Comment:
Did we add a similar error for when a user tries to use the scope and context flags with the env:set command?

Reply:
What happens if they try to use the scope flag but are not on a Pro account or higher?

Reply:
> Did we add a similar error for when a user tries to use the scope and context flags with the env:set command?

Not yet, but we will! `env:set` doesn't support `--context` or `--scope` flags yet, but it's what I'm working on next. 

> What happens if they try to use the scope flag but are not on a Pro account or higher? 

Plan-specific behavior is managed in the API, so the CLI will output any error that the API provides in this case.